src/game/sound_samples.c

```diff
@@ -10,14 +10,92 @@
 #include "system/nth_alloc.h"
 
 // TODO(#863): Sound_samples is not implemented
+// TODO: Volume control?
```

Comment:
Keep in mind that TODOs become titles of issues (see [snitch](https://github.com/tsoding/snitch)). And "Volume control?" is not a good issue title. I suggest something like "SoundSamples does not support volume control".